server/container_status.go

```diff
@@ -69,3 +78,30 @@ func (s *Server) ContainerStatus(ctx context.Context, req *pb.ContainerStatusReq
 	logrus.Debugf("ContainerStatusResponse: %+v", resp)
 	return resp, nil
 }
+
+func (s *Server) getMounts(id string) ([]*pb.Mount, error) {
+	config, err := s.store.GetFromContainerDirectory(id, "config.json")
+	if err != nil {
+		return nil, err
+	}
+	var m rspec.Spec
+	if err = json.Unmarshal(config, &m); err != nil {
```

Comment:
yeah, I noticed that as well, we can filter out stuff like `/proc` and such if you want